features/step_definitions/common_claim_steps.rb

```diff
@@ -102,9 +102,25 @@
 end
 
 When(/^I click "Continue" in the claim form$/) do
+  @claim_form_page.continue_button.click
+  wait_for_ajax
+end
+
+When(/^I click "Continue" in the claim form and move to the '(.*?)' form page$/) do |page_title|
+  original_header = page.first('h2').text
   sleep 3
   @claim_form_page.continue_button.click
   wait_for_ajax
+  using_wait_time(6) do
```

Comment:
Had a look into this and managed to get a non-flickering test using a capybara built-in wait just after the button click but before the wait_for_ajax. 

```
When(/^I click "Continue" in the claim form and move to the '(.*?)' form page$/) do |page_title|
  @claim_form_page.continue_button.click
  page.has_content?(page_title) # wait for next page to load
  wait_for_ajax # wait for fee calc etc. on page
end
```
did not flicker in 20+ tries.

Reply:
👍 Am going to work on a separate branch to see if we can improve this, but for now this is the most non-flaky fix for the features.